int32 id

float64 x
float64 y

float64 distance

# enum: ABS, REL
int16 coord_system

float64 timeout
